src/mixpanel-core.js

```diff
@@ -98,6 +98,7 @@ var DEFAULT_API_ROUTES = {
 var DEFAULT_CONFIG = {
     'api_host':                          'https://api-js.mixpanel.com',
     'api_routes':                        DEFAULT_API_ROUTES,
+    'api_additional_query_params':       '',
```

Comment:
How about instead of passing a preformed query string like `"foo=bar&baz=quux"`, you pass an object `{foo: "bar", baz: "quux"}`. Then above L676 below you can just do `_.extend(data, this.get_config('api_extra_query_params'));`. (Also suggesting a name that's slightly less of a mouthful)
```suggestion
    'api_extra_query_params':            {},
```

Reply:
nice - shall do